module.go

```diff
@@ -45,6 +45,9 @@ const (
 	getPositionsKey           = "getCalibrationPositions"
 	deletePosKey              = "deleteCalibrationPosition"
 	clearCalibrationPositions = "clearCalibrationPositions"
+	autoCalibrateKey          = "autoCalibrate"
```

Comment:
in general, i hate this.
it's so unnecessary, and adds overhead when reading code
if something isn't use more than 2 times, i really don't like this
@Simberific  good example of something to doc